common/genes/02_genes_accessories_hairstyles.txt

```diff
@@ -518,5 +518,43 @@ accessory_genes = {
 			boy = male
 			girl = female
 		}
+		goblin_hairstyles = {
+			index = 19
+			male = {
+				1 = male_hair_sub_saharan_01 #Short Afro
+				1 = male_hair_western_08 #Medium Wavy
+				1 = male_hair_mena_03 #Large Curly
+				1 = male_hair_mena_01 #Curly
+				1 = male_hair_mena_02 #Medium Curls
+				1 = male_hair_mena_04 #Curly Chevelure
+				1 = male_hair_byzantine_01 #Curly Bowl
+				1 = male_hair_byzantine_02 #Roman Fade
+				1 = male_hair_indian_02 #Pulled Back Medium Wavy
+				1 = male_hair_indian_03 #High Bun
+				1 = male_hair_northern_01 #Very Long Wild
+				1 = male_hair_northern_02 #Long Wild
+				1 = male_hair_northern_02
+				1 = male_hair_steppe_01 #Hair Loops
+				1 = male_hair_steppe_02 #Steppe Rider
```

Comment:
I would say these hairstyles give barbaric vibes. Goblins aren't barbaric.

Reply:
Removed in bf8de12a3c69ce1f80cfda13805fdb11da568491.